Inserts.sql

```diff
@@ -1,638 +0,0 @@
--- List all the players for the Red Sox (Full Name, Position, Hits, Throws, Team Abbreviation)
-USE Baseball;
-SELECT CONCAT(firstName, ' ', lastName) AS FullName, position, hits, throws, abbreviation AS "Team Abbreviation"
-FROM players p JOIN teams t ON p.currentTeamId = t.id
-WHERE t.abbreviation = 'BOS';
-
--- Who are the top 5 home run hitters for games played on June 6th? (Full Name, Position, Team Abbreviation, HomeRuns)
-SELECT CONCAT(firstName, ' ', lastName) AS FullName, position, abbreviation, homeRuns
-FROM hittingStats hs JOIN players p on hs.playerId = p.id
-JOIN players p ON hs.playerId = p.id
```

Comment:
You've got two JOINs to player here on the same criteria and using the same aliases. This will definitely throw an error, you should be running these to make sure they work.